lib/Utility/CalendarUtility.dart

```diff
@@ -28,13 +28,13 @@ final kEvents = LinkedHashMap<DateTime, List<Event>>(
 
 
 final _kEventSource = Map.fromIterable(List.generate(50, (index) => index),
-    key: (item) => DateTime.utc(kFirstDay.year, kFirstDay.month, item * 5),
+    key: (item) => TextNote().eventDate,//DateTime.utc(kFirstDay.year, kFirstDay.month, item * 5),
```

Comment:
Why are you creating an instance of Note here? the data should be coming from the db

Reply:
Thank you for pointing that out to me. I corrected that code but please keep in mind this is still in progress. At this point I  am just trying to get things to show up on the calendar.